webapp/src/reducers/index.js

```diff
@@ -193,6 +193,39 @@ const channelSubscriptions = (state = {}, action) => {
     }
 };
 
+const getIssueByKey = (state = {}, action) => {
+    switch (action.type) {
+    case ActionTypes.RECEIVED_JIRA_TICKETS : {
+        const assignee = action.data && action.data.fields && action.data.fields.assignee ? action.data.fields.assignee : null;
+        const ticketData = action.data;
+        const ticketDetails = {
+            assigneeName: assignee && assignee.displayName ? assignee.displayName : '',
```

Comment:
```suggestion
            assigneeName: assignee?.displayName ? assignee.displayName : '',
```

Reply:
check if this works. if yes, update wherever possible

Reply:
No, it will not work as babel is not setup to use optional chaining

Reply:
Then, you should add that just like we did in Gitlab plugin.